.github/workflows/golang.yml

```diff
@@ -35,6 +35,6 @@ jobs:
       - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v7
         with:
-          version: v2.0.2 # this is the golangci-lint version
+          version: v2.1.6 # this is the golangci-lint version
```

Comment:
Updating golangci-lint to get testifylint v1.6.1 that contains this fix:
- https://github.com/Antonboom/testifylint/pull/236

The version fixes an issue where testifylint was reporting false-positives on this PR.